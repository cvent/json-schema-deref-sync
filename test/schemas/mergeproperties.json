{
  "description": "Just a basic schema.",
  "title": "Merge Properties",
  "type": "object",
  "properties": {
    "foo": {
      "$ref": "bar.json",
      "$id": "mergingProperties"
    },
    "bar": {
      "$ref": "bar.json",
      "description": "Merging properties without mutating the defintiion"
    }
  }
}
