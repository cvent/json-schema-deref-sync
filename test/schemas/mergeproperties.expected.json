{
  "description": "Just a basic schema.",
  "title": "Merge Properties",
  "type": "object",
  "properties": {
    "foo": {
      "description": "bar property",
      "type": "boolean",
      "$id": "mergingProperties"
    },
    "bar": {
      "type": "boolean",
      "description": "Merging properties without mutating the defintiion"
    }
  }
}
